src/main/java/com/openmrs/migrator/core/services/impl/PDIServiceImpl.java

```diff
@@ -0,0 +1,54 @@
+package com.openmrs.migrator.core.services.impl;
+
+import com.openmrs.migrator.core.services.PDIService;
+import java.io.InputStream;
+import org.pentaho.di.core.KettleEnvironment;
+import org.pentaho.di.core.Result;
+import org.pentaho.di.core.exception.KettleException;
+import org.pentaho.di.core.logging.LogLevel;
+import org.pentaho.di.job.Job;
+import org.pentaho.di.job.JobMeta;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PDIServiceImpl implements PDIService {
+
+  private static Logger LOG = LoggerFactory.getLogger(PDIServiceImpl.class);
+
+  @Override
+  public boolean runJob(InputStream xmlStream) throws KettleException {
+
+    KettleEnvironment.init();
+
+    JobMeta jobMeta = new JobMeta(xmlStream, null, null);
+
+    Job job = new Job(null, jobMeta);
+    job.setLogLevel(LogLevel.BASIC);
```

Comment:
Lets move this to the config store (this should be a configurable setting for the user) @cfaife

Reply:
Ok I will  implement  it on 85,  it seems appropriate for that

Reply:
sounds good